crypto/s2n_aead_cipher_chacha20_poly1305.c

```diff
@@ -160,7 +160,7 @@ static int s2n_aead_chacha20_poly1305_destroy_key(struct s2n_session_key *key)
 
 static int s2n_aead_chacha20_poly1305_encrypt(struct s2n_session_key *key, struct s2n_blob *iv, struct s2n_blob *aad, struct s2n_blob *in, struct s2n_blob *out)
 {
-    gte_check(in->size, S2N_TLS_CHACHA20_POLY1305_TAG_LEN);
```

Comment:
With this change, what happens if `in->size < S2N_TLS_CHACHA20_POLY1305_TAG_LEN`? I think that means that you have an integer underflow on line 168 (which would be bad).